src/common/pluginloader.js

```diff
@@ -110,6 +116,26 @@ function findCordovaPath() {
     return path;
 }
 
+exports.setModuleInstrumenter = function(moduleID, instrumenter) {
+    if (typeof moduleID !== "string" || typeof instrumenter !== "function") {
+        // ignore
+        return;
+    }
+    moduleInstrumenters[moduleID] = instrumenter;
+}
+
+// Recovers the information from the plugin list metadata and associates each
+// module with its corresponding plugin
```

Comment:
This part is mostly needed for the other party to get module-to-plugin mapping. It adds `plugin` property to each module. Perhaps ideally, this information should be preserved when generating the `cordova_plugins.js` [here](https://github.com/apache/cordova-lib/blob/master/cordova-lib/src/plugman/browserify.js#L132).